amazon-import.user.js

```diff
@@ -0,0 +1,291 @@
+    weight: parseInt(res.weight) * (convertToBB[wtToSIkey] ?? 1),
+    height: parseFloat(height) * (convertToBB[lenghtToSIkey] ?? 1),
+    width: parseFloat(width) * (convertToBB[lenghtToSIkey] ?? 1),
+    depth: parseFloat(depth) * (convertToBB[lenghtToSIkey] ?? 1),
+    date,
+    publisher,
+    format,
+  };
+}
+window.onload = () => {
+  console.log("running amazon import to bb script");
+  if (!document.querySelector("#authorFollowHeading")) {
+    return;
+  }
+  try {
+    // Setting up UI
+    const submitUrl = "https://test.bookbrainz.org/edition/create";
+    const parentEl = document.getElementById("rightCol");
+    const askButton = document.createElement("button");
+    const divContainer = document.createElement("div");
+    askButton.classList.add("bb-btn");
+    askButton.innerText = "Import to BookBrainz";
+    parentEl.insertBefore(askButton, parentEl.children[0]);
+    const expectedOut = {
+      name: "",
+      sortName: "",
+      publisher: "",
+      language: "",
+      pages: "",
+      isbn10: "",
+      isbn13: "",
+      weight: "",
+      height: "",
+      width: "",
+      depth: "",
+      date: "",
+      publisher: "",
+      format: "",
+    };
+
+    let itemDetails;
+    try {
+      itemDetails = scrapeAmz();
+    } catch (err) {
+      console.log("error whilte fetching moving to default, ", err);
+      itemDetails = expectedOut;
+    }
+    console.log("recieved scrape data ", itemDetails);
+    const formHtml = `
+    <h2>Verify Form before submitting</h2>
+<div class="bb-container">
+<h3 class="bb-h3">Edition Entity</h3>
+    <form target="_blank' class="bb-form" action=${submitUrl} method="POST">
+    <label class="bb-flabel" for="bb-name">Name</label>
+    <input class="bb-finput" name="nameSection.name" value="${
+      itemDetails.name
+    }" id="bb-name"/>
+    <label class="bb-flabel" for="bb-sname">Sort Name</label>
+    <input class="bb-finput" name="nameSection.sortName" value="${
+      itemDetails.sortName
+    }" id="bb-sname"/>
+    <label class="bb-flabel" for="bb-language">Language</label>
+    <input class="bb-finput" name="nameSection.language" value="${
+      itemDetails.language
+    }" id="bb-language"/>
+    <label class="bb-flabel" for="bb-isbn13">ISBN 13</label>
+    <input class="bb-finput" name="identifierEditor.t9" value="${
+      itemDetails.isbn13 ?? ""
+    }" id="bb-isbn13"/>
+    <label class="bb-flabchromeel" for="bb-isbn10">ISBN 10</label>
+    <input class="bb-finput" name="identifierEditor.t10" value="${
+      itemDetails.isbn10 ?? ""
+    }" id="bb-isbn10"/>
+    <label class="bb-flabel" for="bb-pub">Publisher</label>
+    <input class="bb-finput" name="editionSection.publisher"  value="${
+      itemDetails.publisher
+    }" id="bb-pub"/>
+    <label class="bb-flabel" for="bb-date">Start date:</label>
+    <input class="bb-finput" name="editionSection.releaseDate" type="date" value=${
+      itemDetails.date
+    } id="bb-date" name="" value="2018-07-22">
```

Comment:
```suggestion
    } id="bb-date">
```
These duplicate attributes are probably left over from testing.